File: code_templates/actions/php/cloneEventNotificationTemplate.php
$template = NULL;
if (isset($_POST["email"])) {
  template = new KalturaEmailNotificationTemplate();
  $recipient = new KalturaEmailNotificationRecipient();
  $recipient->email = $_POST["email"];
  $template->to = new KalturaEmailNotificationStaticRecipientProvider();
  template->to->emailRecipients = array();
  template->to->emailRecipients[0] = $recipient;
} else {
  $template = new KalturaHttpNotificationTemplate();
  $template->url = $_POST["url"];
}
template->name = $_POST["name"];
$template = $client->eventNotificationTemplate->cloneAction(<%- Lucy.code.variable('answers.id') %>, template);
$template = $client.eventNotificationTemplate.updateStatus($template.id, KalturaEventNotificationTemplateStatus::ACTIVE);
<%- Lucy.returnCode('$template') %>

File: code_templates/actions/ruby/cloneEventNotificationTemplate.rb
template = null
if params[:email] then
  template = KalturaEmailNotificationTemplate.new()
  recipient = KalturaEmailNotificationRecipient.new()
  recipient.email = params[:email]
  template.to = KalturaEmailNotificationStaticRecipientProvider.new()
  template.to.email_recipients = [recipient]
else
  template = KalturaHttpNotificationTemplate.new()
  template.url = params[:url]
end
template.name = params[:name]
template = @@client.event_notification_template_service.clone(<%- Lucy.code.variable('answers.id') %>, template)
template = @@client.eventNotificationTemplate.updateStatus(
  template.id,
  KalturaEventNotificationTemplateStatus::ACTIVE)
<%- Lucy.returnCode('template', 4) %> 
